hardware/simulation/simulation.mk

```diff
@@ -56,14 +56,14 @@ endif
 #RULES
 build: $(VSRC) $(VHDR) $(HEXPROGS)
 ifeq ($(SIM_SERVER),)
-	bash -c "trap 'make kill-sim' INT TERM KILL EXIT; make comp"
+	make comp
```

Comment:
why delete this ?

Reply:
Because here is running a build, not a simulation. So there is no simulation stuck to kill.

Reply:
thanks